pages/learn/accounts/enterprise-sso.md

```diff
@@ -0,0 +1,111 @@
+* Terms and Conditions: The terms and conditions that apply to your account will remain the same. However, the company will now be responsible for ensuring compliance with these terms.
+* Data and Privacy: Your personal data associated with the account will be transferred to the company. The company will be responsible for the protection and use of your data in accordance with the existing privacy policy.
+* Consent: By proceeding with this transfer, you confirm that you have the authority to transfer the account to the company and that you consent to the changes outlined above.
+
+By clicking the "Merge account" button, you agree to the above terms. If you have any questions or concerns, please contact our support team before completing the transfer.
+
+### Log in with a SAML account
+
+Once you have enabled SAML on your account, you can log in by following the "Enterprise SSO" login button or the login URL provided by your balenaCloud organization administrator, e.g., `https://dashboard.balena-cloud.com/saml/acme`.
+<!-- TODO: Update to latest screenshot -->
+<img alt="Login page with Enterprise SSO login highlighted." src="/img/common/saml/login-page.png" width="60%">
+
+__Warning:__ Once you have enabled SAML, you can no longer log in using a `password` and should always use the SSO login method.
+
+## FAQs
+
+#### How do I set up an Identity Provider?
+Each SAML Identity Provider (IdP) has its own unique implementation and terminology, which can result in variations in the configuration process. While it is not feasible to provide detailed configuration guidelines for every IdP, we have included example guides for two of the major providers: [Microsoft Entra ID (formerly Azure Active Directory)](/learn/accounts/idp-setup/microsoft-entra-saml-setup/) and [Google Workspace SAML](/learn/accounts/idp-setup/google-workspace-saml-setup/). These examples are designed to help you understand the necessary steps and how to configure your own provider effectively.
+
+#### Can I enforce SAML on all users in my organization?
+It is not yet possible to enforce SAML authentication across your entire organization, but this is a feature we plan to add in the near future.
+
+#### How can I use API keys if SAML users can't create them?
+Currently, SAML authentication users cannot create API keys. If you require API keys for automated processes, we suggest creating a new non-SAML account to act as a "service" account. We plan to add the ability to create fleet and organization-level API keys as a follow-up feature to address this limitation.
+
+<!-- NOTE: we link to this FAQ in the dashboard -->
+#### How do I delete a SAML account?
+To delete a SAML account, you must use the `sdk`. This is only necessary if you are looking to [delete your Idp](#how-do-i-delete-an-identity-provider-in-balenacloud). Removing the user from your IdP will block their balenaCloud access. If you really want to delete your SAML users, execute the following command: 
```

Comment:
Only admins would be able to run this and actually delete. Better to mention that. 

```suggestion
To delete a SAML account, you must use the [balenaSDK](https://docs.balena.io/reference/sdk/node-sdk/). This is only necessary if you are looking to [delete your Idp](#how-do-i-delete-an-identity-provider-in-balenacloud). Removing the user from your IdP will block their balenaCloud access. If you really want to delete your SAML users, execute the following command: 
```